binance.py fetch_positions_risk() - Missing arguments?

https://github.com/ccxt/ccxt/blob/b459dd73c99c744e7f623ed56aa93e646b1f59f8/python/ccxt/binance.py#L5109

I'm getting an error here using the default binance exchange object and calling fetchPositions() with:

```
binance.fetchPositions(params={'type': 'future'})
```

From what I see in the code, fetch_positions_risk() is calling load_leverage_brackets() but not passing `params`.  The error below shouldn't be happening imo if I'm passing `params` to fetch_positions_risk (i.e., it should be passed through so L5109 should be `self.load_leverage_brackets(params=params)`.  Am I completely off here?


```
    def load_leverage_brackets(self, reload=False, params={}):
        self.load_markets()
        # by default cache the leverage bracket
        # it contains useful stuff like the maintenance margin and initial margin for positions
        leverageBrackets = self.safe_value(self.options, 'leverageBrackets')
        if (leverageBrackets is None) or (reload):
            method = None
            defaultType = self.safe_string(self.options, 'defaultType', 'future')
            type = self.safe_string(params, 'type', defaultType)
            query = self.omit(params, 'type')
            if type == 'future':
                method = 'fapiPrivateGetLeverageBracket'
            elif type == 'delivery':
                method = 'dapiPrivateV2GetLeverageBracket'
            else:
>               raise NotSupported(self.id + ' loadLeverageBrackets() supports linear and inverse contracts only')
E               ccxt.base.errors.NotSupported: binance loadLeverageBrackets() supports linear and inverse contracts only

venv/lib/python3.8/site-packages/ccxt/binance.py:4943: NotSupported
```

Happy to open a PR to update this if need be...I've just started using ccxt though so may not be on the right track.

Edit:
Seems the `defaultType` is set to `'spot'` so that's why this is failing:

```
if (leverageBrackets is None) or (reload):
            method = None
            defaultType = self.safe_string(self.options, 'defaultType', 'future')
            type = self.safe_string(params, 'type', defaultType)
            query = self.omit(params, 'type')
            if type == 'future':
                method = 'fapiPrivateGetLeverageBracket'
            elif type == 'delivery':
                method = 'dapiPrivateV2GetLeverageBracket'
            else:
                raise NotSupported(self.id + ' loadLeverageBrackets() supports linear and inverse contracts only')
```

Definitely think `params` should get passed through here...simple fix

--- a/js/binance.js
+++ b/js/binance.js
@@ -5409,7 +5409,7 @@ module.exports = class binance extends Exchange {
             }
         }
         await this.loadMarkets ();
-        await this.loadLeverageBrackets ();
+        await this.loadLeverageBrackets (false, params);
         let method = undefined;
         const defaultType = this.safeString (this.options, 'defaultType', 'future');
         const type = this.safeString (params, 'type', defaultType);
@@ -5442,7 +5442,7 @@ module.exports = class binance extends Exchange {
             }
         }
         await this.loadMarkets ();
-        await this.loadLeverageBrackets ();
+        await this.loadLeverageBrackets (false, params);
         const request = {};
         let method = undefined;
         let defaultType = 'future';

